server/src/models/userModel.js

```diff
@@ -10,7 +10,22 @@ const userSchema = new Schema(
     },
     email: {
       type: String,
-      required: true,
+      lowercase: true,
+      match: [/(.+)@(.+){2,}\.(.+){2,}/, "Please enter a valid email"],
```

Comment:
Can we use the validators package?

Reply:
If there's a plan to use it at multiple places or in all the models, then it makes sense to add it as a dependency and use it.

Reply:
the focus of this pr should be to maintain the uniqueness of the email. even if it's just a sting without a @ sign, let it be. we will make sure the same input doesn't save more than one time.

we will handle validators in different pr since we need in other places too. yes, i think using validator.js is better thank regex.

Reply:
Okay. So should I remove the regex for now?

Reply:
yes, please.